src/rotten-tomatoes/SpoilTomatoes.js

```diff
@@ -2,41 +2,40 @@ class SpoilTomatoes {
 
     constructor(grid) {
         this.grid = grid;
-        this.colLength = 0;
-        this.rowLength = 0;
-        this.getAdjacents = this.getAdjacents.bind(this)
+        this.colLength = this.grid[0].length - 1;
+        this.rowLength = this.grid.length - 1;
     };
 
     getRottenTomatoes() {
-        const locationOfSpoiltTomatoes = [];
+        const spoiltTomatoesLocations = [];
         let numGoodTomatoe = 0;
         for (let row = 0; row < this.grid.length; row++) {
-            this.rowLength = this.grid.length - 1;
             for (let col = 0; col < this.grid[row].length; col++) {
```

Comment:
`this.rowLength`, `this.colLength` can be used here now.

Reply:
I already did that in the constructor, wont it be a repetition of code if i still reassign the same value here.

Reply:
No assign, read.

Reply:
Okay, I have implemented the changes